Jenkinsfile.promote

```
@@ -83,11 +94,44 @@ pipeline {
                 }
             }
         }
-    }
-    post {
-        always {
-            script {
-                cleanImages()
+        stage('Merge PR')
+        {
+            when {
+                expression {
+                    return isRelease() && params.DEPLOY_BUILD_URL != ''
+                }
+            }
+            steps{
+                script {
+                    githubscm.mergePR(deployProperties['kogito-images.pr.link'])
+                }
+            }
+        }
+        stage('Git Tag')
+        {
+            when {
+                return isRelease() && params.DEPLOY_BUILD_URL != ''
+            }
+            steps {
+                script {
+                    checkout(githubscm.resolveRepository("kogito-images", params.GIT_AUTHOR, params.BUILD_BRANCH_NAME, false))
```

Small adjustment to use utils method

```suggestion
                    checkout(githubscm.resolveRepository("kogito-images", getGitAuthor(), getBuildBranchName(), false))
```

---

btw, I think we should checkout before doing the `githubscm.mergePR`.
it makes also more sense to checkout before doing any modification into the repository.
Can you move that checkout to `Merge PR` stage ?

---

@Kaitou786  see this thread